Binaries now exist for 20.04 so use the repo.

jchonig opened the pull request:
<!--
Thank you for contributing to this project!

- This project has a Contributor Code of Conduct: https://voxpupuli.org/coc/
- Please check that here is no existing issue or PR that addresses your problem.
- Our vulnerabilities reporting process is at https://voxpupuli.org/security/

-->
#### Pull Request (PR) description
I checked and upstream binaries exist for 18.04 and at my request they now exist for 20.04, so both versions use the upstream.

I tested this on 20.04 by pointing to my repo in my Puppetfile and it behaves as expected.  I did not test on 18.04

#### This Pull Request (PR) fixes the following issues
n/a

bastelfreak commented:
Hi,
I'm not sure if it's a good idea to change the default value, since that's considered a breaking change. Also in the past I preferred the upstream repos because they were more up2date. What do you think about adding Ubuntu 20 support, but don't change the behavior for the existing operating systems?

jchonig commented:
I updated the PR and case to be just about 20.04.  I also added puppet6 desting to the travis config and the tests now pass.

Puppetlabs doesn't supply puppet5 for 20.04, although that's what's in the Ubuntu repos.  Is there any way to test that?  Are there other test configurations that need to be updated?

Thanks.

Jef
